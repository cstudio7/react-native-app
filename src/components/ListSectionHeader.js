import React from 'react';
import { Text, StyleSheet } from 'react-native';
import Spacing from '../constants/Spacing';

const ListHeaderSection = ({ title }) => (
  <Text style={styles.header}>{title}</Text>
);

const styles = StyleSheet.create({
  header: {
    color: '#979797',
    fontSize: 15,
    lineHeight: 18,
    backgroundColor: '#fff',
    paddingBottom: Spacing.padding,
    marginBottom: 1
  }
});

export default ListHeaderSection;
