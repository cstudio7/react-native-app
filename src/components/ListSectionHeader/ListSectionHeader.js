import React from 'react';
import { View, Text, StyleSheet } from 'react-native';
import { Colors, Spacing, Typography } from '../../constants';

const ListSectionHeader = ({ title }) => (
  <View style={styles.header}>
    <Text style={styles.headerText}>{title}</Text>
  </View>
);

const styles = StyleSheet.create({
  header: {
    backgroundColor: Colors.white,
    paddingLeft: Spacing.padding2,
    paddingRight: Spacing.padding2
  },
  headerText: {
    color: Colors.gray,
    fontSize: Typography.small,
    fontWeight: '500',
    backgroundColor: Colors.white,
    paddingBottom: Spacing.padding,
    marginBottom: 1
  }
});

export default ListSectionHeader;
