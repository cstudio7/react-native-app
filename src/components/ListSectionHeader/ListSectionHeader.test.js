import React from 'react';
import renderer from 'react-test-renderer';
import ListSectionHeader from './ListSectionHeader';

it('renders correctly', () => {
  const tree = renderer.create(<ListSectionHeader title={'A'} />).toJSON();
  expect(tree).toMatchSnapshot();
});
