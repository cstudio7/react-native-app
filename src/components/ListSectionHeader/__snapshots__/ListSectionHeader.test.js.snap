// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "paddingLeft": 16,
      "paddingRight": 16,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "backgroundColor": "#fff",
        "color": "#979797",
        "fontSize": 15,
        "fontWeight": "500",
        "marginBottom": 1,
        "paddingBottom": 8,
      }
    }
  >
    A
  </Text>
</View>
`;
